cli.js

```diff
@@ -8,6 +8,12 @@ const cli = meow(`
 	Usage
 	  $ package-json <name> [version=latest]
 
+	Options
+	  --full-metadata    --full      Output full package metadata
+	  --all-versions     --all       Output all versions
+	  --registry-url     --registry  Registry URL                  [Default: inferred]
+	  --omit-deprecated              Omit deprecated versions      [Default: included]
```

Comment:
```suggestion
	  --no-deprecated              Omit deprecated versions      [Default: included]
```

More in line with CLI conventions.